MODULE.bazel

```diff
@@ -1,25 +1,21 @@
 module(name = "rules_twirl")
 
 bazel_dep(name = "bazel_skylib", version = "1.7.1")
-
-bazel_dep(name = "buildifier_prebuilt", version = "7.3.1", dev_dependency = True)
-
 bazel_dep(name = "rules_java", version = "7.11.1")
-
-bazel_dep(name = "rules_pkg", version = "1.0.1", dev_dependency = True)
-
 bazel_dep(name = "rules_jvm_external", version = "6.4")
 bazel_dep(name = "rules_scala_annex")
 
-rules_scala_annex_version = "lucid_2024-12-06"
+rules_scala_annex_version = "upgrade-to-bazel-8"
```

Comment:
- [ ] Wait until `upgrade-to-bazel-8` is merged and change this to a release tag.